py/server/tests/test_pandas.py

```diff
@@ -165,6 +165,23 @@ def test_to_table_category(self):
         df2 = to_pandas(table)
         self.assertTrue(np.array_equal(df2["A"].values, df2["B"].values))
 
+    def test_to_table_nullable(self):
+        int_array = pd.array([1, 2, None], dtype=pd.Int64Dtype())
+        double_array = pd.array([1.1, 2.2, None], dtype=pd.Float64Dtype())
+        df = pd.DataFrame({"NullableInt64": int_array, "NullableDouble": double_array})
+
+        with self.assertRaises(DHError) as cm:
+            table = to_table(df)
+        self.assertRegex(str(cm.exception), r"TypeError.* NAType")
+
+        int_array = int_array.fillna(NULL_LONG)
+        double_array = double_array.fillna(NULL_DOUBLE)
+        df = pd.DataFrame({"NullableInt64": int_array, "NullableDouble": double_array})
+        table = to_table(df)
```

Comment:
It seems like `to_table` should be transparently handling the NA conversions without the user having to mess with it.

Reply:
I don't oppose to the idea and we have a ticket #2509 for that. To solve would most likely require providing a way for the users to specify replacement values which will be quite a bit more work than what the PR intends to solve.

Reply:
I'm ok if it is another PR.

Reply:
After discussing it with @rbasralian I went ahead and implemented the transparent handling of pd.NA values. No interface change and it should not preclude us from adding user-supplied custom mapping of pd.NA in the future.